rails/es6.md

```diff
@@ -0,0 +1,105 @@
+## [Home](../README.md)
+
+## Summary
+
+Including ES6 in Rails applications can be very useful. Using JavaScript
+packages that are only available/maintained in npm version has traditionally
+been a problem with Rails application. This documents an approach chosen among a
+couple of different options.
+
+We settled upon sprockets-commoner over webpack and browserify because it
+integrates with the Rails asset pipeline and least modifies the natural
+development of JavaScript components.
+
+### Steps to upfit your Rails application
+  * add `'expose window.[component_name]'` to the top of ES6 class files
```

Comment:
Only if you need to reference them somewhere that you can't do a `import` right?